main.go

```diff
@@ -167,6 +164,9 @@ func main() {
 		placementv1.SchemeGroupVersion.WithKind("PlacementRule"): []filteredcache.Selector{
 			{FieldSelector: fmt.Sprintf("metadata.namespace==%s", config.GetDefaultNamespace())},
 		},
+		mchv1.SchemeGroupVersion.WithKind("MultiClusterHub"): []filteredcache.Selector{
```

Comment:
will it be error-ed if there is no `MultiClusterHub`?

Reply:
fixed this by adding check MCH CRD exists before adding the filter cache.